tests/testthat/test-DMA.R

```diff
@@ -0,0 +1,32 @@
+# Test DMA function
+
+# Run example of DMA function
+data(dataMAF)
+data(DEGsmatrix)
+data(dataPRA)
+data(EncodePromoters)
+data(NCG)
+data(dataFilt)
+data(cscape_somatic_output)
+dataDMA <- DMA(dataMAF = dataMAF,
+               dataDEGs = DEGsmatrix,
+               dataPRA = dataPRA,
+               runCscape = FALSE,
+               results_folder = ".")
+
+# Test that output of DMA is as expected
+test_that("output of DMA is a list with 2 elements", {
+  expect_type(dataDMA, "list")
+  expect_named(dataDMA, c("TSG", "OCG"))
+  expect_true(all(sapply(dataDMA, is.character)))
+})
```

Comment:
Ideally yes, I just couldn't figure out how to do this due to the fine balance between having as small as possible example data while still getting meaningful results of the example data. So right now the example data of DMA contains one gene. When I regenerate the example data, the DMA data will be an empty list because the example MAF file and other example data is so small. I couldn't regenerate non-empty DMA data while still keeping the example data files rather small (I tried a lot of different things to generate non-empty DMA example data). 

If I were to add checking the results in this test like in the other tests, it would always fail because the output of the test is empty whereas the example data DMA is non-empty. 

Let me know if you have any ideas on how to solve this

Reply:
let's leave it as is for now, we can see how our reviewer reacts to this